### DISCUSSION TITLE
Can the event command be used in private messages?

### DISCUSSION THREAD

**Discussion by revanapriyandi19** [ArugaZ/whatsapp-bot#481] [General]
import type { Event } from "../../types/command"

export default <Event>{
    execute: async ({ client, message }) => {
        if (!message.fromMe) {
            return await client.sendMessage(message.from, {
                react: {
                    text: "⏳",
                    key: message.key
                }
            })
        }
    }
}

**Comment by arugaz** [comment]
yes, just make sure the filename end with _

```
event_.ts
```

and you can filter the message

```ts
import type { Event } from "../../types/command"

export default <Event>{
  execute: async ({ aruga, message }) => {
    if (!message.isGroupMsg && !message.fromMe) {
      await aruga.sendMessage(message.from, {
        react: {
          text: "⏳",
          key: message.key
        }
      })
    }
  }
}

```

**Reply by ghost** [reply]
add pairing with code ?, its cool
